# Meeting Notes

- **Source audio:** `/home/fschulz/whisper-meeting-notes/dropbox/jobs/20251101-174533-5d1d3604/test_long.mp4`
- **Model:** `large-v3` on `cuda`
- **Generated:** 2025-11-01T10:46:43
- **Participants detected:** Speaker 1

| Start | End | Speaker | Transcript |
|------:|----:|---------|------------|
| 00:00:09.295 | 00:00:09.907 | Speaker 1 | Thank you. |
| 00:01:26.498 | 00:01:26.999 | Speaker 1 | Hi, everyone. |
| 00:01:30.002 | 00:01:30.743 | Speaker 1 | Good morning, Bill. |
| 00:01:31.023 | 00:01:32.765 | Speaker 1 | How are you? |
| 00:01:33.086 | 00:01:34.067 | Speaker 1 | Good to see you, folks. |
| 00:01:35.148 | 00:01:37.311 | Speaker 1 | I was curious, who would actually be at work today? |
| 00:01:40.034 | 00:01:41.236 | Speaker 1 | Oh, with the shutdown, you mean? |
| 00:01:42.117 | 00:01:42.557 | Speaker 1 | Oh, yeah. |
| 00:01:43.318 | 00:01:43.558 | Speaker 1 | Yes. |
| 00:01:43.939 | 00:01:46.181 | Speaker 1 | Yeah, that's for us at Berkeley Lab. |
| 00:01:46.201 | 00:01:49.165 | Speaker 1 | We usually have some, how to say that, |
| 00:01:49.364 | 00:01:54.211 | Speaker 1 | Back up like some reserves, we can, we can keep working for a couple of more weeks. |
| 00:01:54.231 | 00:02:00.621 | Speaker 1 | So we haven't really felt the shutdowns too much yet, but maybe this time we'll see. |
| 00:02:00.641 | 00:02:04.307 | Speaker 1 | It's funny, it's a little different here because we have. |
| 00:02:05.228 | 00:02:10.917 | Speaker 1 | Well, we have a combination military and civilian and contract personnel, but our civilians. |
| 00:02:12.112 | 00:02:37.765 | Speaker 1 | noon yesterday they had to leave so they got all their stuff together and then they put like a letter on their computer saying that can't access it and then they had to walk out the door crazy anyway yeah that's that caused some of the effort ongoing efforts getting cancelled and have to be picked up again anyways but at dirty lap you're like contractors for doe and so |
| 00:02:38.352 | 00:02:42.372 | Speaker 1 | we do have like funds on hand basically to operate longer, but |
| 00:02:43.297 | 00:02:46.141 | Speaker 1 | Don't exactly know the number of weeks, but we'll see. |
| 00:02:46.261 | 00:02:47.442 | Speaker 1 | What about LAN, Migun? |
| 00:02:47.462 | 00:02:48.924 | Speaker 1 | You're also still here, right? |
| 00:02:48.944 | 00:02:49.425 | Speaker 1 | It looks like. |
| 00:02:49.926 | 00:02:53.290 | Speaker 1 | Yeah, I think it's a similar situation. |
| 00:02:53.871 | 00:03:00.159 | Speaker 1 | Unless the government sets down for months, then we'll start seeing the real effect. |
| 00:03:00.639 | 00:03:02.281 | Speaker 1 | There is a DOE office here, though. |
| 00:03:02.382 | 00:03:03.783 | Speaker 1 | I think it's closed, maybe. |
| 00:03:04.625 | 00:03:07.889 | Speaker 1 | But other than that, we got an email one day saying report to duty. |
| 00:03:09.230 | 00:03:09.831 | Speaker 1 | As normal. |
| 00:03:10.352 | 00:03:10.452 | Speaker 1 | Yeah. |
| 00:03:14.009 | 00:03:21.681 | Speaker 1 | Yeah, we had a DOE side office and I forgot if they have new people there because the ones we had before they resigned or they retired early or something earlier this year. |
| 00:03:21.721 | 00:03:25.507 | Speaker 1 | So they were the first ones packing their box and leaving the lab. |
| 00:03:29.253 | 00:03:31.696 | Speaker 1 | But yeah, anyways, they probably have someone now. |
| 00:03:31.716 | 00:03:32.498 | Speaker 1 | So, all right. |
| 00:03:32.518 | 00:03:35.943 | Speaker 1 | It looks like we're getting to the critical mass to get this started. |
| 00:03:36.404 | 00:03:40.109 | Speaker 1 | And I think maybe give it one more minute. |
| 00:03:40.169 | 00:03:40.590 | Speaker 1 | We are like, |
| 00:03:40.958 | 00:04:01.920 | Speaker 1 | eight now maybe then we get 10 that would be good but otherwise if from every institution someone is here then we can probably just start on we got a bit of a delay because we had to cancel last week but it's it's not a problem because we want to come up with that sop in the next few weeks like maybe two to three weeks and then send that around to everyone to chime in one more time |
| 00:04:02.592 | 00:04:11.661 | Speaker 1 | And then at the program update meeting at MIT Lincoln Lab in November, we will present about what we've achieved here in our discussions. |
| 00:04:11.841 | 00:04:15.364 | Speaker 1 | And we are just looking at what we have so far. |
| 00:04:15.464 | 00:04:21.710 | Speaker 1 | It's not going to be that trivial, as we all have guessed already, because there are many things that we can do. |
| 00:04:21.730 | 00:04:24.032 | Speaker 1 | And labs just do it differently. |
| 00:04:25.554 | 00:04:30.118 | Speaker 1 | But the goal here is really to come up with some really baseline workflow that |
| 00:04:30.098 | 00:04:57.675 | Speaker 1 | is easy to adapt for others and they can use it as a reference and if they then show their workflow is actually better then we can we can tweak it or we can we can add more in but we need like a reference workflow that we can compare to and so that's that's the goal mainly here so we'll probably when we present about it in november we will show for each step what the different labs do and then we will have maybe highlighted in a certain color the |
| 00:04:57.655 | 00:05:15.833 | Speaker 1 | baseline workflow we propose based on these discussions and again all of you will hear about it before we present it so you can all chime in and say oh yeah that that makes completely no sense please don't talk about that i mean in this way or like you're missing something here or there so i want to make sure we're all on the same page here and then |
| 00:05:15.813 | 00:05:24.628 | Speaker 1 | Hopefully, moving forward, December, we can start having that reference workflow and maybe every lab can test that and compare it to what they do. |
| 00:05:24.929 | 00:05:27.854 | Speaker 1 | So that would be for this working group the next goal. |
| 00:05:28.294 | 00:05:35.627 | Speaker 1 | And then keep engaging in discussions and talk about how we can optimize that and also thinking more of the |
| 00:05:36.063 | 00:05:40.409 | Speaker 1 | automated platforms, how we can implement that workflow. |
| 00:05:40.790 | 00:05:47.260 | Speaker 1 | And that's something, again, probably more for Lincoln Lab and for Johns Hopkins, to some extent, if they have the platform. |
| 00:05:48.942 | 00:05:52.447 | Speaker 1 | We talked to Krogan Sands, when was that? |
| 00:05:52.908 | 00:06:01.801 | Speaker 1 | A couple of weeks ago, talking about this CDNA, single-strand CDNA that works so well for us, like the additional heating step. |
| 00:06:02.202 | 00:06:04.365 | Speaker 1 | And they said it might be possible in the platform, but |
| 00:06:05.071 | 00:06:11.951 | Speaker 1 | know where they're at at the moment and Nisha mentioned they will probably give it a try and we haven't heard back since then. |
| 00:06:14.098 | 00:06:17.207 | Speaker 1 | All right for today the goal will be to go over |
| 00:06:17.508 | 00:06:21.635 | Speaker 1 | the missing parts of the protocol, which is basically a library preparation. |
| 00:06:22.176 | 00:06:27.044 | Speaker 1 | And as seen in previous discussions, we have different targets. |
| 00:06:27.084 | 00:06:30.710 | Speaker 1 | So it can be DNA, it can be RNA, it can be short read, long read. |
| 00:06:30.730 | 00:06:32.733 | Speaker 1 | So there are different options for library. |
| 00:06:32.833 | 00:06:35.938 | Speaker 1 | There's not just like one library we can do for everything. |
| 00:06:35.918 | 00:07:03.175 | Speaker 1 | so i think we just get started again um with having everyone from like from each institution um one of you talk about how they do the libraries what is you to go where the kids and so on and um what give you the best results uh depending on the use case and then we can make notes and we'll um you know discuss further all right let's get started maybe with uh mrt lingala maybe amy or joe |
| 00:07:04.690 | 00:07:07.053 | Speaker 1 | Sorry, I wasn't on. |
| 00:07:07.554 | 00:07:08.755 | Speaker 1 | I was still on mute. |
| 00:07:09.557 | 00:07:11.980 | Speaker 1 | So Catherine is the ONT expert. |
| 00:07:12.000 | 00:07:13.281 | Speaker 1 | So I'll just give an overview. |
| 00:07:13.302 | 00:07:16.145 | Speaker 1 | And Catherine, feel free to fill in more details. |
| 00:07:17.307 | 00:07:21.733 | Speaker 1 | But for the ONT right now, we're just doing DNA targets. |
| 00:07:21.793 | 00:07:30.124 | Speaker 1 | We tried a couple of the RNA workflows, and they didn't work very well based on the number of reads that we were able to get. |
| 00:07:31.032 | 00:07:35.138 | Speaker 1 | So we use the ligation-based ONT kits for the DNA. |
| 00:07:36.580 | 00:07:50.159 | Speaker 1 | On the Illumina side, we are using the, so we send our samples out for the library prep, but the vendor is using the bead-based Nextera kit to prepare the DNA samples. |
| 00:07:50.800 | 00:07:57.630 | Speaker 1 | And then for the RNA, it is the stranded total RNA prep ligation. |
| 00:07:57.948 | 00:08:04.359 | Speaker 1 | and they pair it with their RIBO0 plus kit to do the rRNA depletion. |
| 00:08:06.663 | 00:08:09.488 | Speaker 1 | And Catherine, yeah, feel free to jump in with additional details. |
| 00:08:16.381 | 00:08:21.510 | Speaker 1 | So maybe a quick question here, as you say, RNA, like that was about direct RNA, correct? |
| 00:08:21.530 | 00:08:22.251 | Speaker 1 | Because we can |
| 00:08:22.754 | 00:08:28.258 | Speaker 1 | for the ONT that that didn't work because maybe it needs a lot of template material or like what were your experiences there? |
| 00:08:29.352 | 00:08:33.756 | Speaker 1 | Yeah, so the direct RNA didn't work well. |
| 00:08:34.317 | 00:08:47.970 | Speaker 1 | So what we found was when we put our wastewater samples through it, we got very few reads out of the wastewater, but we were able to get a lot of reads out of their reference control strand RNA. |
| 00:08:48.451 | 00:08:59.241 | Speaker 1 | So I don't know if it's based on the amount of material or based on the length, if the RNAs from the wastewater are fragmented, degraded, and it doesn't |
| 00:08:59.677 | 00:09:01.179 | Speaker 1 | work well with their kit. |
| 00:09:01.199 | 00:09:06.086 | Speaker 1 | We weren't really able to pinpoint the exact reason why it wasn't working well. |
| 00:09:10.692 | 00:09:12.835 | Speaker 1 | So you mainly talk about RNA. |
| 00:09:13.836 | 00:09:16.219 | Speaker 1 | So when you said the DNA, is that then cDNA? |
| 00:09:16.239 | 00:09:20.005 | Speaker 1 | Or is this an actual metagenome where we use DNA sample? |
| 00:09:20.025 | 00:09:24.831 | Speaker 1 | Or is it from the RNA, you do cDNA, and then you sequence the cDNA on the ONT? |
| 00:09:24.851 | 00:09:25.532 | Speaker 1 | Did you try that? |
| 00:09:26.592 | 00:09:36.529 | Speaker 1 | Oh, so yeah, the ONT DNA that I was mentioning before with the native kit, that's just we extract the DNA and then we prepare the library. |
| 00:09:36.549 | 00:09:50.693 | Speaker 1 | I think, Katherine, correct me if I'm wrong, earlier on we might have tried preparing the cDNA from the RNA and then sequencing, but I think that one was also a little bit difficult as well. |
| 00:09:52.090 | 00:10:07.015 | Speaker 1 | Yeah, we tried using the kyogen whole transcriptome amplification kit and then ended up using the same native flangation kit on the back end to sequence that. |
| 00:10:08.077 | 00:10:09.880 | Speaker 1 | It's a pretty complicated protocol. |
| 00:10:09.860 | 00:10:10.622 | Speaker 1 | overall. |
| 00:10:10.923 | 00:10:17.564 | Speaker 1 | And I'd have to go back and look, but Amy, I believe we had a lot of noise with that protocol as well. |
| 00:10:19.129 | 00:10:20.272 | Speaker 1 | And obviously it's very long. |
| 00:10:21.135 | 00:10:22.840 | Speaker 1 | So we were looking for alternatives. |
| 00:10:28.000 | 00:10:31.707 | Speaker 1 | Did you check your fragment length distribution for ONT? |
| 00:10:31.727 | 00:10:37.279 | Speaker 1 | Did you select certain fragments or so, like longer ones? |
| 00:10:37.359 | 00:10:44.052 | Speaker 1 | Or is there anything you did in addition to the standard protocol, especially for the DNA? |
| 00:10:45.247 | 00:10:53.901 | Speaker 1 | Yeah, so out of wastewater, our DNA just tends to be anywhere from 600 to 1,000 base pairs. |
| 00:10:54.181 | 00:10:57.466 | Speaker 1 | So we don't do any sort of fragmentation or size selection. |
| 00:10:57.987 | 00:10:59.510 | Speaker 1 | That seems to work fairly well for us. |
| 00:11:00.611 | 00:11:04.858 | Speaker 1 | But out of the WTA, obviously those fragments are extremely long. |
| 00:11:04.878 | 00:11:07.322 | Speaker 1 | So we did use the cobaris. |
| 00:11:07.302 | 00:11:11.459 | Speaker 1 | to fragment that before putting it onto the flow cell. |
| 00:11:11.780 | 00:11:17.363 | Speaker 1 | And that seemed to actually help to have the shorter fragments rather than, I think it was |
| 00:11:18.575 | 00:11:19.737 | Speaker 1 | I'd have to go back and look. |
| 00:11:19.777 | 00:11:23.463 | Speaker 1 | It was very long, definitely over 2000 base pairs long. |
| 00:11:23.644 | 00:11:29.354 | Speaker 1 | So we fragmented to about 550 to 600 base pairs to run that. |
| 00:11:29.474 | 00:11:33.421 | Speaker 1 | And that helped improve the poor clogging that I think we were experiencing. |
| 00:11:33.841 | 00:11:37.748 | Speaker 1 | But for the DNA and the RNA, we don't do any fragmentation. |
| 00:11:45.085 | 00:12:12.662 | Speaker 1 | So out of curiosity, now that you have data from both the ONT and the short read, and you had also some process controllers, so what worked better for you in like, let's say, per base, per gigabase sequence or so, or like per megabase sequence, if you like, calculate that, was it more sensitive to do ONT or short read, like consider, like in the light of the sequencing efforts that was performed? |
| 00:12:14.515 | 00:12:16.857 | Speaker 1 | It's maybe a difficult question without slides, I agree. |
| 00:12:16.877 | 00:12:22.943 | Speaker 1 | But I would just be curious from everyone to hear a bit about their experiences in terms of sensitivity. |
| 00:12:22.983 | 00:12:27.868 | Speaker 1 | Because again, that's one of the goals that we can detect what we want to detect. |
| 00:12:27.968 | 00:12:31.012 | Speaker 1 | And if you don't have a lot of reads, maybe anyone's missing it. |
| 00:12:35.476 | 00:12:44.505 | Speaker 1 | Yeah, I would say we have been doing some spike studies and very preliminary results. |
| 00:12:44.688 | 00:12:46.953 | Speaker 1 | are suggesting the Illumina is better. |
| 00:12:47.013 | 00:12:52.905 | Speaker 1 | But the problem is, as you mentioned, with the normalization to the base pairs. |
| 00:12:52.965 | 00:12:55.851 | Speaker 1 | So I need to look more carefully into that. |
| 00:12:56.513 | 00:13:00.862 | Speaker 1 | But I think definitely cost-wise, Illumina is better for us. |
| 00:13:00.942 | 00:13:03.928 | Speaker 1 | So that's something we can say definitively. |
| 00:13:07.132 | 00:13:07.493 | Speaker 1 | Thanks. |
| 00:13:09.259 | 00:13:10.743 | Speaker 1 | Sorry. |
| 00:13:10.924 | 00:13:14.655 | Speaker 1 | We're also seeing that, Amy, in the work we've done here. |
| 00:13:15.679 | 00:13:18.407 | Speaker 1 | Sorry, I didn't mean to interrupt, but just for context. |
| 00:13:21.509 | 00:13:40.497 | Speaker 1 | Maybe when we have that meeting in November, if everybody could have on their presentation, if it fits onto the slides, just like a slide that shows a bit on the sensitivity, like normalize somehow to the read depths and the sequence effort and like that, that we all get a better idea with slides and with the data backing this up. |
| 00:13:40.477 | 00:13:47.164 | Speaker 1 | And I think for now, for the SOP, I think this is OK, because we're anyways looking at the different possible ways of how we do it. |
| 00:13:47.204 | 00:13:51.369 | Speaker 1 | And we cannot, let's say, exclude ONT right now, because it's nice to deploy ONT. |
| 00:13:51.449 | 00:13:52.370 | Speaker 1 | It's small and everything. |
| 00:13:53.831 | 00:13:59.037 | Speaker 1 | But yeah, anyways, it would be good to see as much data as possible, what would support either method. |
| 00:14:01.840 | 00:14:04.122 | Speaker 1 | Are there any other questions for MIT Lincoln Lab? |
| 00:14:14.412 | 00:14:21.078 | Speaker 1 | Then let's go to Lionel. |
| 00:14:22.479 | 00:14:30.946 | Speaker 1 | So we have Andy here who's going to, but we haven't done a lot of OID sequencing and the sequencing that we've done mostly has been in Element. |
| 00:14:31.627 | 00:14:35.050 | Speaker 1 | And Andy, do you want to summarize that? |
| 00:14:35.750 | 00:14:43.457 | Speaker 1 | Yeah, so yeah, like Magun just said, the vast majority of our sequencing with the wastewater samples has been using the Element ABD sequencer. |
| 00:14:43.808 | 00:14:48.652 | Speaker 1 | which if you're not familiar, it's similar to Illumina in that it's a short read sequencer. |
| 00:14:48.692 | 00:14:56.519 | Speaker 1 | And it's basically the same sequencing depth, similar kits, low output, medium output, high output. |
| 00:14:58.621 | 00:14:59.302 | Speaker 1 | Yeah, it works well. |
| 00:14:59.322 | 00:15:00.123 | Speaker 1 | The quality is good. |
| 00:15:01.003 | 00:15:08.370 | Speaker 1 | I can't really, I don't know if I can speak to comparing Element to Illumina for wastewater. |
| 00:15:08.730 | 00:15:13.815 | Speaker 1 | Like if we're seeing much of a difference, we haven't really done those studies. |
| 00:15:14.285 | 00:15:24.562 | Speaker 1 | But I find that using the element of ED with their Elevate enzymatic library prep kit makes really good sequencing library from wastewater samples. |
| 00:15:25.003 | 00:15:29.470 | Speaker 1 | And we've processed wastewater samples several different ways using different kits. |
| 00:15:30.251 | 00:15:33.036 | Speaker 1 | And all the libraries always look really good. |
| 00:15:34.973 | 00:15:36.335 | Speaker 1 | That's really all I've got to say. |
| 00:15:36.355 | 00:15:47.729 | Speaker 1 | We've done some experimental work with ONT, but nothing to really compare the ONT with the element, just because I don't know of how you would really compare. |
| 00:15:47.769 | 00:15:57.240 | Speaker 1 | The depth of sequencing for the element is so much greater than ONT that we definitely see more organisms like viruses and stuff with that. |
| 00:15:59.743 | 00:16:00.104 | Speaker 1 | What else? |
| 00:16:00.404 | 00:16:01.285 | Speaker 1 | I think that's all. |
| 00:16:01.940 | 00:16:10.432 | Speaker 1 | And for the ONT sequencing, Andy, when we did some anecdotal studies, are they direct RNA sequencing or are they cDNA? |
| 00:16:11.473 | 00:16:11.854 | Speaker 1 | It's cDNA. |
| 00:16:12.114 | 00:16:16.661 | Speaker 1 | Yeah, I've never done direct RNA sequencing with ONT. |
| 00:16:16.681 | 00:16:16.821 | Speaker 1 | Okay. |
| 00:16:16.881 | 00:16:20.646 | Speaker 1 | But Amy, in MIT, you guys have done the direct RNA sequencing? |
| 00:16:20.666 | 00:16:21.087 | Speaker 1 | Is that right? |
| 00:16:22.309 | 00:16:22.429 | Speaker 1 | Okay. |
| 00:16:22.509 | 00:16:26.074 | Speaker 1 | And then in terms of the fragment length, are you guys getting... |
| 00:16:26.780 | 00:16:33.769 | Speaker 1 | comparable, maybe not comparable to DM, but how bad does it look in terms of fragment length for the direct RNA sequencing? |
| 00:16:37.192 | 00:16:39.215 | Speaker 1 | That I'll have to double check. |
| 00:16:39.255 | 00:16:49.508 | Speaker 1 | I think our direct RNA results were just so bad that we didn't look more into them, but I can double check the length and get back to you on that. |
| 00:16:50.029 | 00:16:56.678 | Speaker 1 | For your direct RNA sequencing that you do from wastewater samples with the ONT, does it do the poly capture? |
| 00:16:56.698 | 00:16:58.120 | Speaker 1 | Is that how the libraries are prepped? |
| 00:16:59.045 | 00:17:04.892 | Speaker 1 | Yes, so the direct RNA kit itself, I believe, is based on a poly capture. |
| 00:17:04.932 | 00:17:12.582 | Speaker 1 | So to make sure that we were getting everything into that poly, we had to add poly to all of our RNA. |
| 00:17:12.662 | 00:17:20.292 | Speaker 1 | And so it just became like, I think, a lengthy process where there's a lot of opportunities for loss or bias. |
| 00:17:21.554 | 00:17:25.879 | Speaker 1 | Maybe that is what contributed also to the poor results in the end. |
| 00:17:31.411 | 00:17:39.489 | Speaker 1 | Maybe one question, and I think we have something when Ava will talk about what we try to do, but what about single-stranded DNA viruses? |
| 00:17:39.569 | 00:17:46.704 | Speaker 1 | Anyone thought of those and like think which kit you use, how that actually impacts the recovery of single-stranded DNA viruses? |
| 00:17:46.744 | 00:17:51.715 | Speaker 1 | Because we had some in our spike in panel and Ava might have more information here, but |
| 00:17:52.100 | 00:18:01.440 | Speaker 1 | We found that some kids are really good at missing single-stranded DNA viruses, while others seem to be better in recovering this type of target. |
| 00:18:02.482 | 00:18:07.913 | Speaker 1 | Eva, that's only single-stranded DNA, or that wasn't the main problem sometimes, is that correct? |
| 00:18:09.757 | 00:18:10.699 | Speaker 1 | Yes, mostly. |
| 00:18:11.810 | 00:18:16.214 | Speaker 1 | So, so, well, anyone else also problematic if. |
| 00:18:16.234 | 00:18:19.637 | Speaker 1 | Like, the viruses in the wastewater, I kind of like a damaged. |
| 00:18:20.638 | 00:18:22.800 | Speaker 1 | So, it could be problem that we don't catch it. |
| 00:18:23.300 | 00:18:27.804 | Speaker 1 | Because it's so yeah, it also kind of help when we. |
| 00:18:27.824 | 00:18:30.206 | Speaker 1 | Like, tested the single standard DNA protocols. |
| 00:18:31.607 | 00:18:34.410 | Speaker 1 | Kind of like an improved. |
| 00:18:35.511 | 00:18:40.635 | Speaker 1 | Lightly everything the single standard. |
| 00:18:41.881 | 00:18:49.757 | Speaker 1 | So we recently published something with a data set from an EPA group in Cincinnati. |
| 00:18:49.777 | 00:18:57.753 | Speaker 1 | So they had this data set that they did not just on the influent, but they were looking at the activated slugs and the effluent. |
| 00:18:58.194 | 00:19:03.805 | Speaker 1 | And they had done both metagenomes and the MDA sequencing, the rolling circle amplifications. |
| 00:19:03.785 | 00:19:17.423 | Speaker 1 | We did find that I think if you use the MDAs or the ruling circle type amplification, you do end up getting a lot more single-stranded DNA viruses than from metagenomes. |
| 00:19:17.690 | 00:19:39.291 | Speaker 1 | So I do feel like that's another workflow to add to your sample, but it does help that if you do these aggressive MDAs or maybe not as aggressive, then you're able to recover a lot more single-stranded DNA viruses than from the shotgun methods. |
| 00:19:39.791 | 00:19:45.857 | Speaker 1 | Shotgun do recover them as well, but I think not at the level of the MDAs. |
| 00:19:47.204 | 00:19:50.378 | Speaker 1 | I actually tested one of those protocols. |
| 00:19:51.282 | 00:19:54.717 | Speaker 1 | It was working, but it did a lot of lag. |
| 00:19:56.570 | 00:20:22.408 | Speaker 1 | problems that I have problem with the yield after and there was like a lot of difficulties with it but right now there is several kids and protocols that actually don't need it they are like a commercially based that I just start with the single standard DNA with the damaged DNA and they managed to like create it it worked with the splints oligo it catches the |
| 00:20:22.388 | 00:20:48.801 | Speaker 1 | dna the single standard one and that it like start to create a library it's much more easier it's more efficient and like we just started to test it but it looks pretty good from the results we are getting from the illumina right now so yeah and do you end up combining the libraries from the from the from the protocol that you just described to the shotgun or do you do it separately |
| 00:20:48.950 | 00:20:57.349 | Speaker 1 | I tested it on the sample that we have results from an extra kit and also I did I prepared with a single standard DNA protocol. |
| 00:20:57.870 | 00:21:00.757 | Speaker 1 | So I think we have those results already. |
| 00:21:00.797 | 00:21:02.942 | Speaker 1 | Like the comparison. |
| 00:21:05.082 | 00:21:21.162 | Speaker 1 | I think there was a, and we should probably pull up that slide, but there was only one case where the normal library kit for, that was for Illumina that worked better, but for almost all the cases, the single-stranded kit worked better, even for non-single-stranded targets. |
| 00:21:21.803 | 00:21:25.607 | Speaker 1 | And if I remember correctly, so what was the company producing this? |
| 00:21:25.847 | 00:21:28.150 | Speaker 1 | That was from the Bay Area startup. |
| 00:21:28.170 | 00:21:29.512 | Speaker 1 | Yeah, it's from Santa Cruz. |
| 00:21:29.712 | 00:21:31.174 | Speaker 1 | It's Claret Bioscience. |
| 00:21:33.128 | 00:21:38.593 | Speaker 1 | And the name of the kid is SRY or something like an acronym. |
| 00:21:39.053 | 00:21:41.215 | Speaker 1 | We have it somewhere written down. |
| 00:21:41.575 | 00:21:43.077 | Speaker 1 | S-R-S-L-Y. |
| 00:21:44.518 | 00:21:45.719 | Speaker 1 | S-R-S-L-Y. |
| 00:21:45.739 | 00:21:47.100 | Speaker 1 | Seriously, kid. |
| 00:21:49.622 | 00:21:49.843 | Speaker 1 | Yeah. |
| 00:21:50.063 | 00:21:57.129 | Speaker 1 | Is it possible, as we discussed some of these protocols, to also include the time it takes to get through them? |
| 00:21:57.850 | 00:22:03.034 | Speaker 1 | So for the initial baseline, I don't think it matters, but when we start thinking about |
| 00:22:03.217 | 00:22:06.820 | Speaker 1 | how to automate, I think the time is going to be really important. |
| 00:22:06.840 | 00:22:09.919 | Speaker 1 | So it might help us to triage what we can and can't do. |
| 00:22:13.898 | 00:22:15.321 | Speaker 1 | That's a wonderful point, Joe. |
| 00:22:16.243 | 00:22:18.447 | Speaker 1 | I'll give you an interesting case in point. |
| 00:22:18.487 | 00:22:24.058 | Speaker 1 | One of the things that we're evaluating is, of course, you know, twist hybrid capture versus the other kits out there. |
| 00:22:24.098 | 00:22:31.613 | Speaker 1 | And that's a two day prep, you know, going through the, you know, selective enrichment versus library prep. |
| 00:22:31.633 | 00:22:36.042 | Speaker 1 | And so having a good understanding and back to your point, Fred, that. |
| 00:22:36.022 | 00:22:43.094 | Speaker 1 | We're thinking about a robust platform that could be as fully automated as possible with as simplistic approach as possible. |
| 00:22:43.174 | 00:22:44.657 | Speaker 1 | And ideally, that could be OT. |
| 00:22:45.158 | 00:22:50.787 | Speaker 1 | But if it's simply not going to work in terms of sensitivity, what are the down selects in terms of selecting other methods? |
| 00:22:51.409 | 00:22:54.013 | Speaker 1 | So, yeah, definitely appreciate the conversation. |
| 00:22:57.048 | 00:23:03.122 | Speaker 1 | So I know we're getting off-topic here, but did anyone try those latest Benchtop Illumina? |
| 00:23:03.322 | 00:23:04.565 | Speaker 1 | I forgot the name of it. |
| 00:23:04.605 | 00:23:08.594 | Speaker 1 | We've seen it earlier this year, I think, a Benchtop Illumina sequencer. |
| 00:23:09.636 | 00:23:10.578 | Speaker 1 | MySeqi100? |
| 00:23:11.400 | 00:23:13.625 | Speaker 1 | Yeah, probably this one, yeah. |
| 00:23:13.645 | 00:23:15.850 | Speaker 1 | Do you have them, or do you have it? |
| 00:23:17.804 | 00:23:18.285 | Speaker 1 | Yeah, we do. |
| 00:23:18.325 | 00:23:24.373 | Speaker 1 | I would say it's definitely an upgrade from the previous MySeq. |
| 00:23:24.693 | 00:23:47.743 | Speaker 1 | Not to the NextSeq level, the NextSeq 2000 here, but I'd say it's a good intermediate device that, again, not to get off topic, but I would say that if folks are starting with the environmental surveillance capability in general, that should be their ground zero device for NGS just to get their foot in the door and then build from there. |
| 00:23:49.816 | 00:23:51.639 | Speaker 1 | At least that's what we're recommending to people. |
| 00:23:56.488 | 00:24:01.096 | Speaker 1 | At Lincoln Laboratory, we just bought a Promethean ONT. |
| 00:24:02.639 | 00:24:13.478 | Speaker 1 | I think we're at least initially finding that it's superior to the Minion, and you're not really trading off much in terms of size, but you get a lot more depth. |
| 00:24:17.103 | 00:24:18.326 | Speaker 1 | Yeah, that's a great point. |
| 00:24:18.346 | 00:24:20.852 | Speaker 1 | So I think Eva can maybe talk about our protocols. |
| 00:24:20.893 | 00:24:24.361 | Speaker 1 | But we've been using Depromethion mainly because just of the throughput. |
| 00:24:24.401 | 00:24:29.053 | Speaker 1 | And it's much cheaper, the flow cells, compared to Minion per base. |
| 00:24:30.029 | 00:24:36.579 | Speaker 1 | But yeah, maybe Eva, you can summarize our approaches and then we can go to Bill and others. |
| 00:24:36.599 | 00:24:40.926 | Speaker 1 | There's like a lot of things that we are still testing. |
| 00:24:42.528 | 00:24:46.935 | Speaker 1 | Mainly the RNA, we are processing to cDNA with the standard kits. |
| 00:24:47.416 | 00:24:50.721 | Speaker 1 | Just as Farid mentioned, there is the |
| 00:24:50.701 | 00:24:55.727 | Speaker 1 | 95 degree denaturation pre-stat before we start to see DNA. |
| 00:24:55.747 | 00:25:02.155 | Speaker 1 | It really helps with the yield at the end, and it helps with the double-stranded RNA stuff in it. |
| 00:25:04.077 | 00:25:16.793 | Speaker 1 | And then we, still as the main standard, we used Nexra, but we started to test also the single-stranded DNA kits even for the RNA. |
| 00:25:17.819 | 00:25:18.982 | Speaker 1 | So we use it for both. |
| 00:25:19.944 | 00:25:27.202 | Speaker 1 | I think it was originally made for like an archeological kind of sample that are really damaged. |
| 00:25:27.522 | 00:25:30.670 | Speaker 1 | So it's also really helping for the wastewater. |
| 00:25:30.690 | 00:25:31.913 | Speaker 1 | I think it's really good. |
| 00:25:33.496 | 00:25:38.081 | Speaker 1 | Or it's for like a short read sequencing. |
| 00:25:38.141 | 00:25:44.909 | Speaker 1 | For the long read, we started with testing protocol that is called R2C2. |
| 00:25:45.810 | 00:25:49.274 | Speaker 1 | It's before we start to use the ligation kit. |
| 00:25:50.315 | 00:25:51.777 | Speaker 1 | And it helps with two things. |
| 00:25:52.478 | 00:25:57.183 | Speaker 1 | First of all, we don't need to have, the input doesn't need to be that high. |
| 00:25:57.223 | 00:26:03.030 | Speaker 1 | And the second, it makes the, it's more, how to say it, |
| 00:26:04.141 | 00:26:33.473 | Speaker 1 | uh like a proofread that it doesn't make like a it helps with uh i'm missing the word today so it's more accurate so we have uh like at least like five and five rounds um of amplification um yeah if you're polya like we have like this dna of the split which makes that um circular fragment then and then we can do five passes or more of like similar to what um |
| 00:26:33.453 | 00:26:40.090 | Speaker 1 | That increases the accuracy to something like Q40 or Q50 99.995 or so. |
| 00:26:40.110 | 00:26:45.083 | Speaker 1 | It should be similar to Illumina, maybe slightly better even. |
| 00:26:46.328 | 00:27:07.154 | Speaker 1 | Again, that's a lot of work in progress, and there's one paper out on it, but the person who published that, she was hired at JGI to develop that further for metagenomics and metatranscriptomics, and so she worked with Eva on some of this, basically trained Eva on it as well, so we can use this R2C2 approach. |
| 00:27:07.895 | 00:27:16.285 | Speaker 1 | There's still a little bit of tweaking needed, and we are still carefully assessing all the results, and we had some good data on the wastewater from R2C2, and |
| 00:27:16.265 | 00:27:22.891 | Speaker 1 | compared to the poly-A based direct sequencing, like from the cDNA. |
| 00:27:22.911 | 00:27:28.496 | Speaker 1 | So we didn't do direct RNA, because at JGI they focus a lot on direct RNA, and they got good results. |
| 00:27:28.876 | 00:27:33.681 | Speaker 1 | But I think it's from plants or some other samples where they have a lot of template material. |
| 00:27:35.002 | 00:27:40.367 | Speaker 1 | So that is not the case for us, where we have very poor template material oftentimes. |
| 00:27:41.247 | 00:27:43.009 | Speaker 1 | And it's difficult to get in that range. |
| 00:27:43.049 | 00:27:46.272 | Speaker 1 | I forgot what it was, 500 nanogram or microgram |
| 00:27:46.252 | 00:27:57.210 | Speaker 1 | So we didn't even want to start that, so we just work on cDNA there when we do the RNA. |
| 00:27:57.230 | 00:27:59.894 | Speaker 1 | Now, for the processing times, Eva, what do you think? |
| 00:27:59.934 | 00:28:04.181 | Speaker 1 | Like, how does that compare the... Yeah, that's the problem. |
| 00:28:04.201 | 00:28:12.495 | Speaker 1 | It still takes, because there is the Gibson assay, and it takes overnight, so it's usually two to three days to finish the protocol right now. |
| 00:28:12.762 | 00:28:17.308 | Speaker 1 | Another thing is that the original protocol it's for poly targets only. |
| 00:28:17.328 | 00:28:30.444 | Speaker 1 | But I was testing it also with like a random priming cDNA already made that we had in the lab and it seems working too. |
| 00:28:30.464 | 00:28:36.492 | Speaker 1 | There were just like slightly needed to switch some oligos and it like it seems it's working. |
| 00:28:37.602 | 00:28:51.207 | Speaker 1 | So we need to prepare the protocol for a DNA, because there are also some problems with the oligos that we need to figure out how to attach it to actually create the circularization and yeah. |
| 00:28:52.756 | 00:29:10.849 | Speaker 1 | So the idea is to amplify differently than using the poly-A as the splint, but to get everything basically, not just the poly, the viruses that are polyadenylated or the transcripts that are polyadenylated, but use random, is it hexameras or like random, probably similar to what we do for whole genome amplification? |
| 00:29:11.707 | 00:29:15.811 | Speaker 1 | And then I'll use this for the circularization. |
| 00:29:17.252 | 00:29:19.995 | Speaker 1 | So we have just got data back. |
| 00:29:20.015 | 00:29:21.536 | Speaker 1 | We're analyzing it at the moment. |
| 00:29:21.576 | 00:29:22.998 | Speaker 1 | First glimpse, it looks good. |
| 00:29:23.038 | 00:29:31.085 | Speaker 1 | But still need to look more carefully into it in terms of what's actually fragment length, because it seems to be slightly lower compared to the other approach. |
| 00:29:31.346 | 00:29:33.227 | Speaker 1 | There could be some issues with this. |
| 00:29:33.307 | 00:29:39.714 | Speaker 1 | But what we now can maybe do is assemble it, because we have covered more of those viruses if they're present. |
| 00:29:39.774 | 00:29:41.275 | Speaker 1 | And then let's say 30 can be. |
| 00:29:41.255 | 00:30:05.625 | Speaker 1 | um rna virus if you just do poly a maybe we get 5kb of it if you're lucky or something and maybe maybe 10. we had to fall for like sasko with two words with the peg bio isostic but then um it's usually from the it's a three three prime and we have the poly a tail and then um we don't capture the whole thing but with the random priming there is a chance that we get multiple fragments and we can then assemble it |
| 00:30:05.605 | 00:30:29.571 | Speaker 1 | and that's all with ont with little amount of templates so if this method works it could be really good uh moving forward and but but yeah it's maybe in november we can provide some detailed update on this um once we've analyzed it thoroughly and then from the processing time as ava said r2c2 is slightly it takes slightly more time at the moment and not sure if there's any way of we could improve it um |
| 00:30:31.002 | 00:30:42.022 | Speaker 1 | Yeah, there's a problem with the enzymes that it takes very long time to process, to circularize the things and actually to create the triangles, like several ones to spin it. |
| 00:30:43.124 | 00:30:50.057 | Speaker 1 | Also, I forgot to mention that we don't use any RNA depletion, like a bacterial one. |
| 00:30:50.105 | 00:30:56.131 | Speaker 1 | Because we figured out when we pre-filter the samples before it goes through the whole processing, we don't use it at all. |
| 00:30:56.612 | 00:31:01.737 | Speaker 1 | Because with this, we were losing a lot of material with the cleaning and everything. |
| 00:31:01.878 | 00:31:04.561 | Speaker 1 | And it was just a counterproductive. |
| 00:31:04.581 | 00:31:07.283 | Speaker 1 | So it's a fast time to do this step. |
| 00:31:09.726 | 00:31:15.272 | Speaker 1 | Yeah, we started out with that step because I think at CGI, they use it by default, the RNA depletion. |
| 00:31:15.332 | 00:31:16.213 | Speaker 1 | But then... |
| 00:31:16.430 | 00:31:24.967 | Speaker 1 | We realized for our samples, as Eva said, that the pre-spin helped a lot to reduce bacteria, the bacteria in there. |
| 00:31:25.188 | 00:31:27.913 | Speaker 1 | And again, we split the samples. |
| 00:31:27.933 | 00:31:28.795 | Speaker 1 | We talked about it last time. |
| 00:31:28.815 | 00:31:36.531 | Speaker 1 | We do the part with the DNA, what we spin down versus the supernatant, which will process further for the viruses. |
| 00:31:37.878 | 00:31:45.891 | Speaker 1 | Eva, for the other kits, like the processing time, let's say the single strand DNA kit, what's the processing time there compared to the Nextera? |
| 00:31:45.911 | 00:31:52.742 | Speaker 1 | Because Nextera is super straightforward, but are there any additional, does it take longer because we do these repair steps and everything? |
| 00:31:52.762 | 00:31:53.243 | Speaker 1 | Not really. |
| 00:31:53.283 | 00:31:54.705 | Speaker 1 | It's really easy. |
| 00:31:54.725 | 00:31:57.810 | Speaker 1 | I think it was even slightly shorter than the Nextera. |
| 00:31:59.242 | 00:32:12.171 | Speaker 1 | What I'm testing now for RNA targets, just to do the cDNA, like the single strand, and it's just enough to create like the library from it. |
| 00:32:12.370 | 00:32:18.637 | Speaker 1 | The problem is that I have less material like that because I have only one strand, but it's still working. |
| 00:32:19.077 | 00:32:20.079 | Speaker 1 | So I save time. |
| 00:32:20.219 | 00:32:27.066 | Speaker 1 | I like an hour at least to not to go through like a second strand cDNA creation. |
| 00:32:27.106 | 00:32:33.313 | Speaker 1 | So it's like, it's really open and it's like really easy to use and they are still working on it. |
| 00:32:33.774 | 00:32:37.618 | Speaker 1 | As I know, like a clever science, like how to do it. |
| 00:32:38.628 | 00:32:54.141 | Speaker 1 | I'm still using Covaris for like shearing the sample, but now they have also enzymatic version, but the problem is it's made for like a more concentrated sample, like the higher yields. |
| 00:32:54.161 | 00:32:59.252 | Speaker 1 | So I was like talking to them and they said they are afraid that if we use like just a |
| 00:33:00.447 | 00:33:05.638 | Speaker 1 | small amount of like an input material that they would share it too much. |
| 00:33:05.658 | 00:33:06.760 | Speaker 1 | So they didn't recommend it. |
| 00:33:06.780 | 00:33:13.474 | Speaker 1 | They said we should use the calories right now, but maybe, yeah, they are still working on it. |
| 00:33:13.494 | 00:33:14.636 | Speaker 1 | They are improving their kids. |
| 00:33:14.656 | 00:33:16.380 | Speaker 1 | So like it's a new company. |
| 00:33:18.267 | 00:33:25.639 | Speaker 1 | So, for the automated platform, like, from Krogensens, if there are the shearings, that would be a problem? |
| 00:33:25.739 | 00:33:29.064 | Speaker 1 | Or is that, like, Joe and Amy, is this something they're already doing? |
| 00:33:29.164 | 00:33:31.789 | Speaker 1 | Or would enzymatic be the solution, probably? |
| 00:33:32.009 | 00:33:41.504 | Speaker 1 | They're not shearing on purpose, but I think their nucleic acid extraction is pretty harsh. |
| 00:33:41.724 | 00:33:44.629 | Speaker 1 | So, I think they're getting a lot of shearing based off of that. |
| 00:33:49.891 | 00:34:10.988 | Speaker 1 | That reminds me something we haven't discussed when we talk about nucleic acid extraction and something maybe we should be discussing in a follow-up meeting is like, and this brings us out of the library, but it's thinking of bacterial pathogens and spores because that's something we thought like we also have microsporidia or like things that are difficult to lyse and to open up, like to get the nucleic acid effectively out. |
| 00:34:10.968 | 00:34:16.599 | Speaker 1 | So maybe one of the subsequent meetings, you should talk a little bit about best practices here, like how we can be as thorough as possible. |
| 00:34:16.759 | 00:34:19.464 | Speaker 1 | Because, again, spores are a big problem, I guess. |
| 00:34:19.525 | 00:34:29.584 | Speaker 1 | Eva, was there anything to add to the library prep from our side for, let's say, |
| 00:34:30.188 | 00:34:39.036 | Speaker 1 | OMT for the Prometheon, like any, like if you do the non R2C2 protocol, like just the direct poly A based selection. |
| 00:34:39.597 | 00:34:43.280 | Speaker 1 | We have a tester because we never had enough material to actually try it. |
| 00:34:44.722 | 00:34:47.124 | Speaker 1 | But with R2C2, we managed. |
| 00:34:48.005 | 00:34:57.313 | Speaker 1 | I had the sample that had like two nanogram per microliter and I was able to create a library from it for OMT and it was enough. |
| 00:34:57.333 | 00:34:57.934 | Speaker 1 | So yeah. |
| 00:34:59.804 | 00:35:02.466 | Speaker 1 | Yeah, we had good results, and I will present it in November. |
| 00:35:02.907 | 00:35:12.555 | Speaker 1 | Maybe I had it already in the last program update meeting, but then we got better recovery per base pair with the ONT than compared to Illumina. |
| 00:35:12.575 | 00:35:14.557 | Speaker 1 | And maybe that was just in one experiment. |
| 00:35:14.677 | 00:35:18.761 | Speaker 1 | So we look at the next batch of data that comes back if we can reproduce that. |
| 00:35:18.821 | 00:35:21.083 | Speaker 1 | But it looked quite promising. |
| 00:35:21.103 | 00:35:23.886 | Speaker 1 | But I think this was the poly-A-based R2C2 still. |
| 00:35:23.926 | 00:35:26.688 | Speaker 1 | So there might have been some bias of the library. |
| 00:35:26.708 | 00:35:29.811 | Speaker 1 | Perhaps maybe it worked extremely well for certain mammalian pathogens or so. |
| 00:35:29.791 | 00:35:31.773 | Speaker 1 | like our spike-ins. |
| 00:35:32.635 | 00:35:35.158 | Speaker 1 | Anyways, that looks really promising there. |
| 00:35:35.258 | 00:35:41.445 | Speaker 1 | But again, that was from one sequencing run, and we will see it for the next one we're working on right now, if we can reproduce that. |
| 00:35:45.691 | 00:35:53.380 | Speaker 1 | All right, I think if that's all from our side, then we can maybe go to Bill to chat more about the library product. |
| 00:35:53.400 | 00:35:54.401 | Speaker 1 | I'm so sorry, Frederik. |
| 00:35:54.902 | 00:35:56.764 | Speaker 1 | I'm just getting pulled to another call. |
| 00:35:57.065 | 00:35:58.887 | Speaker 1 | I wish I would have spoke earlier. |
| 00:36:00.284 | 00:36:18.915 | Speaker 1 | Okay, so again, what I'll do is I'll summarize from what we're doing at our team here and also our partners at USAF, Sam, but we've kind of tried to do a cross, like a two-dimensional matrix, thinking about different platforms themselves, like the MySeq, the NexE 2000, MinION, GridION. |
| 00:36:19.736 | 00:36:24.604 | Speaker 1 | Now we're getting the PacBioVega and the NovaSeqX, but also the different prep platforms. |
| 00:36:24.584 | 00:36:27.528 | Speaker 1 | perhaps either within the platforms or across platforms. |
| 00:36:27.609 | 00:36:37.784 | Speaker 1 | And just broad characterization, yes, absolutely what we're seeing with ONT is that the cDNA, whenever you have low template, is the approach that we found with higher efficacy. |
| 00:36:38.405 | 00:36:47.459 | Speaker 1 | In general, I would love to talk to you about the results where you did this cross comparison between ONT and Illumina and found that ONT had a better sensitivity. |
| 00:36:47.499 | 00:36:48.700 | Speaker 1 | That's fascinating to me. |
| 00:36:49.241 | 00:36:51.705 | Speaker 1 | Usually we're seeing exactly the opposite. |
| 00:36:51.685 | 00:36:58.556 | Speaker 1 | especially when we start to incorporate some of these hybrid capture, VRCAPSEQ, BATCAPSEQ capabilities as well. |
| 00:36:59.076 | 00:37:01.520 | Speaker 1 | So, again, damn it. |
| 00:37:02.021 | 00:37:04.585 | Speaker 1 | This is like the kind of science that I like to do. |
| 00:37:05.707 | 00:37:18.006 | Speaker 1 | But, again, what I'll do is, you know, especially for this update in November, I'll try to synthesize what we have here and what we found, at least the first steps across these different preps. |
| 00:37:18.186 | 00:37:20.089 | Speaker 1 | Would that be useful for you and the group here? |
| 00:37:21.402 | 00:37:22.003 | Speaker 1 | Okay. |
| 00:37:22.103 | 00:37:24.886 | Speaker 1 | Okay, wonderful. |
| 00:37:25.087 | 00:37:28.651 | Speaker 1 | And again, I'm sorry that I have to drop this damn shutdown. |
| 00:37:28.831 | 00:37:29.572 | Speaker 1 | It's killing me. |
| 00:37:29.592 | 00:37:41.807 | Speaker 1 | But the things that you guys have discussed have been really, really good for me to hear because it's helping for us to be a sanity check because we've encountered some of the same problems. |
| 00:37:42.168 | 00:37:43.149 | Speaker 1 | So thank you so, so much. |
| 00:37:43.530 | 00:37:45.051 | Speaker 1 | And again, sorry, my apologies. |
| 00:37:45.352 | 00:37:46.834 | Speaker 1 | I'll check in as soon as I can. |
| 00:37:47.695 | 00:37:48.456 | Speaker 1 | Completely no problem. |
| 00:37:48.476 | 00:37:49.797 | Speaker 1 | Thanks for joining today. |
| 00:37:49.817 | 00:37:51.139 | Speaker 1 | It was great to have you here. |
| 00:37:53.330 | 00:38:10.242 | Speaker 1 | and all right let's see do we have a another institution here johns hopkins did you did you do any of the um library prep or all right um |
| 00:38:10.576 | 00:38:19.972 | Speaker 1 | I'll have to check with the other task lead, Tim, but he's on the bioinformatics side and doing more of the analysis. |
| 00:38:20.172 | 00:38:28.346 | Speaker 1 | I don't know as far as the experimental work that you all have been referring to, how much of that has been done at APL to date. |
| 00:38:28.786 | 00:38:33.394 | Speaker 1 | Natalie is currently out of the country, which is why I'm stepping in, but I'll get back to you. |
| 00:38:35.602 | 00:38:51.990 | Speaker 1 | sounds good perfect on i think for today as we we got through all these steps and maybe something if there's anything else for from crack and sense like for the calypso and any additional information we would need um to factor in when we |
| 00:38:51.970 | 00:38:54.174 | Speaker 1 | We're working on this SOP. |
| 00:38:54.234 | 00:39:00.206 | Speaker 1 | Maybe, Joe, you can remind us one more time, what's the current status there? |
| 00:39:00.346 | 00:39:03.192 | Speaker 1 | Did you see data coming out of it, and how did that look like? |
| 00:39:04.875 | 00:39:07.741 | Speaker 1 | What are the current steps on the platform that are being used? |
| 00:39:09.264 | 00:39:10.907 | Speaker 1 | Yeah. |
| 00:39:12.187 | 00:39:22.506 | Speaker 1 | We were working pretty intensely with the Calypso from about December of last year through May of this year. |
| 00:39:22.526 | 00:39:34.988 | Speaker 1 | And we were able to get the instrument to run through all the fluidic steps, but we were getting extremely low yield of nucleic acids on the back end. |
| 00:39:34.968 | 00:39:42.057 | Speaker 1 | And I think we determined that the fluidic chip that Kraken Sense designed was just not it. |
| 00:39:42.578 | 00:39:47.244 | Speaker 1 | It was more suitable for PCR, but it was not suitable for sequencing. |
| 00:39:47.304 | 00:39:56.957 | Speaker 1 | So they have gone through back to the drawing board to try to re-engineer the fluidic chip to get higher nucleic acid capture. |
| 00:39:57.437 | 00:39:59.440 | Speaker 1 | That's what they've been working on this summer. |
| 00:40:01.023 | 00:40:17.749 | Speaker 1 | In addition to that, Lincoln Laboratory is working on designing a, we're trying to get the Promethean workflow, sorry, I apologize, the Promega workflow to work on the Opentrons. |
| 00:40:17.729 | 00:40:22.515 | Speaker 1 | And so we are working through a design right now to try to get that to work on the Opentrons. |
| 00:40:22.575 | 00:40:25.798 | Speaker 1 | And this is kind of like a backup to the Calypso. |
| 00:40:26.319 | 00:40:32.646 | Speaker 1 | But it would also be really useful in terms of processing samples if we can get that automated to work. |
| 00:40:33.167 | 00:40:40.075 | Speaker 1 | So we haven't run many samples through the Calypso just because of that nucleic acid yield was just really low. |
| 00:40:42.569 | 00:41:07.159 | Speaker 1 | The steps are, they draw a sample in, they spin it down to remove the large particulates, then they treat it with guanidine for the extraction, and then they run it through their fluidic chip, where the chip does the capture and the wash, and then the |
| 00:41:07.139 | 00:41:21.655 | Speaker 1 | you know after after the washing steps it does the illusion off of the uh the capture zone so again they they need to kind of they're focusing on trying to uh expand that so also to add to this um |
| 00:41:22.495 | 00:41:31.815 | Speaker 1 | With this particular project, the DITRA project, Kraken Sense, at least the date, has not been funded directly by DITRA. |
| 00:41:32.396 | 00:41:40.032 | Speaker 1 | So they were, you know, we gave them some money to, we bought the first instrument, the kind of the... |
| 00:41:40.552 | 00:41:42.656 | Speaker 1 | the beta version of the instrument. |
| 00:41:43.717 | 00:41:57.360 | Speaker 1 | There's currently a request for a proposal that's out, so DITRA is trying to solicit automated technologies to try to get through this agnostic sequencing. |
| 00:41:57.981 | 00:42:03.150 | Speaker 1 | I think Kraken Sense is definitely applying to it, but I think there's other companies that are applying to it as well. |
| 00:42:03.130 | 00:42:04.494 | Speaker 1 | I don't know how it's going to shake out. |
| 00:42:04.594 | 00:42:15.925 | Speaker 1 | I don't know who is evaluating that, but we'll see hopefully in the November or so timeframe who is going to get funded to actually like really concentrate on this effort. |
| 00:42:17.558 | 00:42:18.159 | Speaker 1 | That sounds good. |
| 00:42:18.179 | 00:42:19.040 | Speaker 1 | Thanks for these updates. |
| 00:42:19.701 | 00:42:20.442 | Speaker 1 | That's very helpful. |
| 00:42:20.723 | 00:42:23.787 | Speaker 1 | Very curious about one more thing for that calypso. |
| 00:42:23.907 | 00:42:27.793 | Speaker 1 | As you said, it kind of comes from the PCR world, and it works really well there. |
| 00:42:27.813 | 00:42:37.867 | Speaker 1 | They showed it during the pandemic and so on, but would it be easier to refactor it for something like a hybridization probe capture or so, like thinking of this twist comprehensive panel or something? |
| 00:42:38.188 | 00:42:46.840 | Speaker 1 | It wouldn't be for absolutely novel pathogen detection, but it can detect novel pathogens, let's say, if it's a new species or even part of a genus that's maybe on that |
| 00:42:46.820 | 00:42:48.062 | Speaker 1 | capture panel. |
| 00:42:48.342 | 00:42:51.686 | Speaker 1 | So there is definitely a lot of potential in the twist comprehensive panel. |
| 00:42:51.706 | 00:42:56.733 | Speaker 1 | And I think a lot of collaborators we have here in the Bay Area, they're using this by default. |
| 00:42:57.234 | 00:43:04.784 | Speaker 1 | And Miguel, I'm not sure if you're using the comprehensive panel as well, or like for your public codes, like surveillance efforts. |
| 00:43:04.804 | 00:43:12.574 | Speaker 1 | And that's, but because... We're just doing qPCR now, yeah. |
| 00:43:15.085 | 00:43:22.816 | Speaker 1 | Yeah, so, you know, Kraken Sense claims their instrument's very adjustable and flexible and they can add this and that in. |
| 00:43:23.738 | 00:43:27.263 | Speaker 1 | Any change to their current protocol is going to be R&D. |
| 00:43:27.443 | 00:43:28.525 | Speaker 1 | That's how I see it. |
| 00:43:28.745 | 00:43:35.575 | Speaker 1 | So if you want to do the twist panel, I'm not saying it can't be done on their instrument, but they're going to have to do some R&D to kind of figure it out. |
| 00:43:38.238 | 00:43:38.819 | Speaker 1 | That makes sense. |
| 00:43:39.700 | 00:43:46.651 | Speaker 1 | I was just thinking, anything amplification-based, if they have a problem in the end with the yield or so, there's not enough material to sequence for the library. |
| 00:43:47.812 | 00:43:57.046 | Speaker 1 | Anyhow, if there's amplification step, it could really help, something like the twist comprehensive panel or where we use the probes with amplification step. |
| 00:43:57.267 | 00:43:59.129 | Speaker 1 | I'm not sure, going back to that MDA part. |
| 00:43:59.169 | 00:44:06.220 | Speaker 1 | So we tried, actually, MDA on some of the spiking targets, but in the wastewater. |
| 00:44:06.200 | 00:44:08.404 | Speaker 1 | It's just they didn't really help us too much. |
| 00:44:08.444 | 00:44:17.921 | Speaker 1 | It amplified too much of bacterial random stuff, what we didn't want, where we actually designed our probes without their specific for certain conserved regions in viruses. |
| 00:44:18.001 | 00:44:22.669 | Speaker 1 | But then, like, yeah, anyways, wastewater is just extremely complex. |
| 00:44:22.790 | 00:44:28.039 | Speaker 1 | But yeah, thinking of what you mentioned with the single-stranded probes, |
| 00:44:28.019 | 00:44:29.263 | Speaker 1 | protocol with the MDA. |
| 00:44:29.323 | 00:44:37.928 | Speaker 1 | And we said we tested it all, but it was not like a traditional where we amplify really a lot out of the sample. |
| 00:44:40.355 | 00:44:48.584 | Speaker 1 | We, for some reason, in Amplify, just like certain things, and we were losing some of them, even our spike-ins in it. |
| 00:44:48.604 | 00:44:54.450 | Speaker 1 | It was just like very selective for some reason, but I tested it only like on two samples, so yeah. |
| 00:44:54.470 | 00:45:07.965 | Speaker 1 | Maybe if I played with it more, I would figure out what was wrong, but it was kind of like surprising that I get just like... And were these samples concentrated somehow before, or...? |
| 00:45:08.182 | 00:45:10.825 | Speaker 1 | No, I just used the one I wanted to like compare it. |
| 00:45:10.885 | 00:45:17.773 | Speaker 1 | So I used one sample for like a several different protocols and yeah, this one wasn't like really good one. |
| 00:45:18.914 | 00:45:19.014 | Speaker 1 | Okay. |
| 00:45:22.978 | 00:45:23.179 | Speaker 1 | All right. |
| 00:45:23.199 | 00:45:26.883 | Speaker 1 | Is there anything to add to what we discussed for the library prep today? |
| 00:45:28.384 | 00:45:30.747 | Speaker 1 | Anything we're missing or we should still talk about? |
| 00:45:34.186 | 00:45:46.605 | Speaker 1 | I don't otherwise we can write it up and then we send it around and set in the beginning we'll try to send this out earlier this time, so we get more feedback and we come up to a patch together that. |
| 00:45:47.927 | 00:45:53.055 | Speaker 1 | So P and showing like different labs do that on potentially differently and like what's the. |
| 00:45:53.035 | 00:46:17.035 | Speaker 1 | more straightforward way to go and still based on discussions without without seeing this on the paper right now but um and also what shows that for the platform like with the for the calypso or similar platform where we were with the promega um the question is um with backman that for us it seems still very straightforward way of doing the work is is that correct about there was the |
| 00:46:18.854 | 00:46:24.823 | Speaker 1 | So yeah, anyways, this might be a contender for the baseline, but I'm not sure. |
| 00:46:24.943 | 00:46:28.288 | Speaker 1 | Mikko, do you have a VACMAN at LANL? |
| 00:46:28.688 | 00:46:30.871 | Speaker 1 | I don't think it's so expensive, actually. |
| 00:46:33.916 | 00:46:34.376 | Speaker 1 | VACMAN? |
| 00:46:35.137 | 00:46:38.001 | Speaker 1 | VACMAN, V-A-C-M-A-N. |
| 00:46:38.022 | 00:46:41.527 | Speaker 1 | If the Promega could, yeah. |
| 00:46:41.547 | 00:46:43.950 | Speaker 1 | Oh, no, we don't. |
| 00:46:44.892 | 00:46:45.733 | Speaker 1 | Because that's the other thing. |
| 00:46:45.773 | 00:46:50.059 | Speaker 1 | If you develop that workflow that we should, like a reference workflow, it would be good. |
| 00:46:50.099 | 00:46:53.824 | Speaker 1 | Every lab has it available so they can actually test it. |
| 00:46:55.106 | 00:47:02.055 | Speaker 1 | And so, Megan, do you think this is something you could get in case we propose that? |
| 00:47:02.075 | 00:47:03.818 | Speaker 1 | Most likely you're going to propose that probably. |
| 00:47:04.519 | 00:47:05.480 | Speaker 1 | Yeah, yeah. |
| 00:47:05.500 | 00:47:06.401 | Speaker 1 | I mean, we can look into it. |
| 00:47:06.922 | 00:47:09.926 | Speaker 1 | You say it's not that expensive, right? |
| 00:47:09.946 | 00:47:11.288 | Speaker 1 | Yeah, we'll look into it. |
| 00:47:12.635 | 00:47:14.803 | Speaker 1 | The VacMan's just like a vacuum manifold, right? |
| 00:47:15.907 | 00:47:16.751 | Speaker 1 | That you use with that kit. |
| 00:47:17.734 | 00:47:17.995 | Speaker 1 | Yeah. |
| 00:47:18.156 | 00:47:21.067 | Speaker 1 | If you can do the kyogen preps, you could do that. |
| 00:47:21.348 | 00:47:23.978 | Speaker 1 | So I'm sure you have the capability. |
| 00:47:27.198 | 00:47:40.617 | Speaker 1 | All right, then I don't want to take up more of your time today, and we'll get 15 minutes back, and we'll connect again, potentially one more time before the November meeting, in case we have open questions, something we need to discuss further. |
| 00:47:41.138 | 00:47:50.151 | Speaker 1 | Otherwise, see you all in Boston, or most of you in Boston in November, and we'll follow up by email individually. |
| 00:47:50.131 | 00:47:54.686 | Speaker 1 | Yeah, just one more thing to add with that November meeting with the government shutdown. |
| 00:47:54.766 | 00:47:57.174 | Speaker 1 | I'm not I'm not entirely sure what's going to happen. |
| 00:47:57.294 | 00:47:58.659 | Speaker 1 | So, you know, just stand by. |
| 00:48:00.003 | 00:48:03.735 | Speaker 1 | Well, we'll send information as soon as we we hear |
| 00:48:06.466 | 00:48:10.151 | Speaker 1 | So it could be that we do it remotely, or just postpone it? |
| 00:48:10.872 | 00:48:12.294 | Speaker 1 | Well, so I don't know. |
| 00:48:12.475 | 00:48:15.359 | Speaker 1 | On the government side, I don't know what their status is. |
| 00:48:15.419 | 00:48:17.722 | Speaker 1 | I'm assuming Steve is not working. |
| 00:48:18.724 | 00:48:20.206 | Speaker 1 | But all of us obviously are. |
| 00:48:20.266 | 00:48:22.749 | Speaker 1 | So I think it's probably still important to meet. |
| 00:48:23.070 | 00:48:31.562 | Speaker 1 | Maybe it does turn virtual, but I think we could still meet without Steve there, especially if Bill Koalas is available. |
| 00:48:31.542 | 00:48:40.975 | Speaker 1 | So I haven't had a discussion with Steve yet about what to do, but I plan on trying to do that, and we'll have a contingency plan if the government is still shut down. |
| 00:48:46.302 | 00:48:47.323 | Speaker 1 | Perfect. |
| 00:48:53.992 | 00:48:54.112 | Speaker 1 | Okay. |
| 00:48:54.132 | 00:48:54.753 | Speaker 1 | Thanks, everybody. |
| 00:48:54.773 | 00:48:56.275 | Speaker 1 | Thank you. |
| 00:48:56.415 | 00:48:56.575 | Speaker 1 | Bye. |
| 00:48:57.757 | 00:48:57.837 | Speaker 1 | Bye. |
